config.py

```diff
@@ -14,8 +14,8 @@ class StageConfig(Config):
     ENV = 'stage'
     TESTING = True
 
-
 class DevelopmentConfig(Config):
     DEBUG = True
     ENV = 'development'
-    TESTING = True
\ No newline at end of file
+    TESTING = True
+    SECRET_KEY = 'default_secret'
```

Comment:
Where is this used?

Reply:
It's used by Flask's session feature to secure the cookie!